apps/crt0.s

```diff
@@ -13,11 +13,11 @@ _start:
 
 # Resets
 reset_handler:
-  li  x31 , 0x2028 # CR_CURSOR_H Address
+  la  sp, _stack # Stack pointer
+  li  x31 , 0x7028     # CR_CURSOR_H Address
```

Comment:
Not sure this is where we want to reset the cursor.

We should have a function that can move the cursor to any point - useful for writing to screen on different locations.

Reply:
We have to force the cursor reset action and I haven't seen anywhere else to force it, so here it is. As for the function it exist.